.env.example

```diff
@@ -1,30 +1,34 @@
-# ---------------------------------
+#################################
 #   Environment configuration 
-# ---------------------------------
+#################################
 
-# Local: Ganache-cli
+# Ethereum node and Contract: Local, Ganache-cli
 NODE_URL=ws://localhost:8545
 STABLE_COIN_CONTRACT_ADDRESS=0x855d1c79Ad3fb086D516554Dc7187E3Fdfc1C79a
 
-# Rinkeby
+# Ethereum node and Contract: Rinkeby
+#   https://github.com/gnosis/dex-contracts/blob/master/networks.json
 #NODE_URL=https://rinkeby.infura.io/v3/<your-api-key>
 #STABLE_COIN_CONTRACT_ADDRESS=0x9046451F7cF124c1d7d1832F76F5e98a33D1610E
 
-# Production
+# Ethereum node and Contract: Production
+#   https://github.com/gnosis/dex-contracts/blob/master/networks.json
 #NODE_URL=https://mainnet.infura.io/v3/<your-api-key>
 #STABLE_COIN_CONTRACT_ADDRESS=<not-deployed-yet>
 
-# WEB: Front end. It can be any instance of this web https://github.com/gnosis/dex-react
+# Web - It can be any instance of this web:
+#   https://github.com/gnosis/dex-react
 WEB_BASE_URL=https://dex-react
 
-# ---------------------------------
+
+#################################
 #   Other config
-# ---------------------------------
+#################################
 
-# Telegram
+# Telegram: Use the /newbot command to create a new bot using "The Bot Father"
+#   https://t.me/BotFather
 TELEGRAM_TOKEN=1234567:asdfghjkl
 TELEGRAM_CHANNEL_ID=@dFusionPoC
 
-#PK=my-private-key
-#MNEMONIC=my super secret mnemonic
-NODE_URL=http://localhost:8545
```

Comment:
I understand this is an example, but is NODE_URL not used anymore?

Reply:
yes, `NODE_URL` but is part of the env var. I've removed it from there